src/pages/Accounts/AddWalletWithLedger.tsx

```diff
@@ -306,15 +308,38 @@ export function AddWalletWithLedger() {
                 <LedgerConnector
                   onSuccess={onSuccess}
                   onTroubleshoot={() => setStep(Step.Troubleshoot)}
+                  checkIfWalletExists
                 />
               </Stack>
               <Stack sx={{ p: 2, mb: 2, rowGap: 1 }}>
-                <Button
-                  disabled={!hasPublicKeys}
-                  onClick={() => setStep(Step.Name)}
-                >
-                  {t('Next')}
-                </Button>
+                <Stack sx={{ flexDirection: 'row', columnGap: 2 }}>
+                  <Stack sx={{ width: '50%' }}>
+                    <Tooltip
+                      title={t(
+                        'Clicking the cancel button will close the tab and open the extension for you. If the extension doesn’t open automatically, please open it manually.'
+                      )}
+                    >
+                      <Button
+                        color="secondary"
+                        onClick={() => {
+                          browser.action.openPopup();
```

Comment:
This is a great addition! Let's also do it on the success screen, when users click "Close" 🎉